.gitignore

```diff
@@ -17,4 +17,5 @@ package-lock.json
 /specs/data/migration/output
 /packages/**/build
 *.tsbuildinfo
-3d-tiles-tools-*.tgz
\ No newline at end of file
+3d-tiles-tools-*.tgz
+.idea
```

Comment:
I use WebStorm for TypeScript & JavaScript so I added this ignore so my project files don't accidentally get checked in.